pkg/api/short_url.go

```diff
@@ -57,6 +57,11 @@ func (hs *HTTPServer) redirectFromShortURL(c *models.ReqContext) {
 		return
 	}
 
+	// Failure to update last seen at should still allow to redirect
```

Comment:
I think when referring to the field `LastSeenAt`, it would be better to write it out in its field form IE `LastSeenAt` or `last_seen_at`, just so it doesn't confuse the sentence in the comment or error log

Reply:
```suggestion
	// Failure to update LastSeenAt should still allow to redirect
```